File: frontend/src/elements/PriceTag.js
import React from "react";

const PriceTag = ({ price }) => {
  return (
    <div className="PriceTag flex justify-center items-center relative w-9 h-9">
      <img src="pricetag.png" alt="" className="absolute w-9 h-9" />
      <h3 className="absolute text-white font-semibold text-sm">₹{price}</h3>
    </div>
  );
};

export default PriceTag;

File: src/src/components/Menu/price_tag.py
import reflex as rx


def PriceTag(price) -> rx.Component:
    return rx.chakra.box(
        rx.chakra.image(src="images/pricetag.png", class_name="w-9 h-9"),
        rx.chakra.text(f"₹{price}", class_name="absolute text-white font-semibold text-sm"),
        class_name="PriceTag flex justify-center items-center relative w-9 h-9",
    )
